packages/website/docs/adding-recent-searches.md

```diff
@@ -68,6 +68,51 @@ Since the `recentSearchesPlugin` reads from [`localStorage`](https://developer.m
   openOnFocus={true}
 />
 
+## Transforming recent searches
+
+If you use Autocomplete as an entry point to a search page, you can turn recent searches into links:
+
+```js
+const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
+  // ...
+  transformSource({ source }) {
+    return {
+      ...source,
+      getItemUrl({ item }) {
+        return `/search?q=${item.query}`;
+      },
+      templates: {
+        item(params) {
+          const { item } = params;
+          return (
+            <a className="aa-ItemLink" href={`/search?q=${item.query}`}>
+              {source.templates.item(params)}
+            </a>
+          );
+        },
+      },
+    };
+  },
+});
+```
+
+If you use Autocomplete on your instant search page, you can plug some logic with `onSelect`:
```

Comment:
```suggestion
If you use Autocomplete on an [Instant Search](https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/js/) page, you can modify the Instant Search query state when a user selects an autocomplete item with [`onSelect`](sources/#onselect):
```

Reply:
I wanted to avoid mentioning InstantSearch specifically. This might be better, wdyt?

```suggestion
If you use Autocomplete on the same page as your main search and want to avoid reloading the full page when an item is selected, you can modify your search query state when a user selects an item with [`onSelect`](sources/#onselect):
```

Reply:
Got it. This looks good!

Reply:
https://github.com/algolia/autocomplete/pull/449/commits/420629b85f8eb215d5a811810743b0d8213ed881